dashboard/app/utils/clickhouse.ts

```diff
@@ -250,6 +250,19 @@ export async function countFeedbacksForMetric(
   metric_config: MetricConfig,
 ): Promise<number> {
   const metric_table_name = getMetricTableName(metric_config);
+
+  // Special handling for demonstration feedback which doesn't use metric_name
+  if (metric_config.type === "demonstration") {
+    const query = `SELECT COUNT() AS count FROM ${metric_table_name}`;
```

Comment:
You're not filtering for the function name. I don't think we have this in the table. We can do a join here for now (~ select from demonstrations, inner join with inferences, where function name = ...). Then open an issue to add this column and/or a materialized view. Right now this returns higher count than inferences with fixtures.

Reply:
I don't think we need to filter for function name here. We do this below in `countDemonstrationDataForFunction` and this information is already displayed